### ISSUE TITLE
Rudder & Throttle Up/Down doesn't work for keyboard useres

### ISSUE THREAD

**Issue by ICanFly256** [heclak/community-a4e-c#310]
**Describe the bug**

Hello,
So I am still a relatively newer user of DCS World and I am using the A-4 Skyhawk as my first full-fidelity model. Anyway, I originally had an older model installed (I think V 1.4) and I recently have downloaded the most recent update for the community A-4. When I loaded back up the game and proceeded through the start-up sequence, I noticed I was not able to increase my throttle to proceed to taxi to the runway. The only buttons that worked to increase throttle were the - and + on my number pad instead of the "page up" and "page down". Anyway, after I managed to increase throttle with the other key binding I then noticed that the rudder was not responsive at all. Even with the rudder trim all the way to the right/left, it did not move at all and proceeds to go straight. 

If you need more details I will be sure to find a way to supply evidence to you. When I had the older model installed everything worked completely fine. The rudder went each way accordingly and the throttle was able to be increased or decreased using page up or down.

Thanks.

THIS WAS FOR BETA TEST! LATEST PRE RELEASE

**Expected behavior**

**Screenshots**

**Software Information (please complete the following information):**
 - DCS Version: [e.g. DCS 2.5.4.30386 Open Beta]
 - A-4E Version [e.g. v1.3]

**Additional context**

**Comment by 08jne01** [comment]
You need to enable nosewheel steering failing that please check your installation with the installation instructions. If the issue persists the  open another defect report with a description on how to reproduce.
